Content.Server/Botany/SeedPrototype.cs

```diff
@@ -208,12 +214,19 @@ public class SeedData
 
     [DataField("plantIconState")] public string PlantIconState { get; set; } = "produce";
 
-    [DataField("bioluminescent")] public bool Bioluminescent;
+    [DataField("mandragoraSound")]
```

Comment:
Is there any particular reason why this is called a `mandragoraSound` and not a `sound`?

Reply:
Just for case if plant would have other sounds in the future .
Like for harvesting or instrument usage.

Reply:
Just make it obvious when it fires because i have no clue.

Reply:
yeah , make sence ,my bad. sorry its my first time , i promise next PR will be 4 times better , spoiler gona make plant scanner like in ss13.